src/ResourceManager/AzureStackAdmin/Commands.AzureStackAdmin/Nouns.cs

```diff
@@ -22,42 +22,47 @@ internal static class Nouns
         /// <summary>
         /// The prefix for cmdlet names.
         /// </summary>
-        private const string Prefix = "AzureRM";
+        private const string Prefix = "AzS";
```

Comment:
@bganapa should the prefix be changed to `Azs` to avoid confusion with the capital S?